src/node_trace_events.cc

```diff
@@ -63,7 +63,7 @@ void NodeCategorySet::Enable(const FunctionCallbackInfo<Value>& args) {
   NodeCategorySet* category_set;
   ASSIGN_OR_RETURN_UNWRAP(&category_set, args.Holder());
   CHECK_NE(category_set, nullptr);
-  auto categories = category_set->GetCategories();
+  const auto& categories = category_set->GetCategories();
```

Comment:
thank you, I consistently forget that the `const` and `&` are still required with `auto`.